### PR TITLE
Fedora 23 fixes

### PR DESCRIPTION
**Pull request by mvollmer**
This should also give us a 'stable' f23 image.

### CODE REVIEW

**Comment by mvollmer** [comment]
This should leave only one failure:

```
ERROR: testBasic (__main__.TestMultiMachineKeyAuth)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./check-multi-machine-key", line 144, in testBasic
    "2048 SHA256:gwSyNJuAvsZXlsv+zQNten2XaCe4cw6Gbqw2tmFTBRA /home/admin/.ssh/identity (RSA)",
  File "./check-multi-machine-key", line 85, in test_keys
    b.wait_js_func("(function (expected) { return window.test_result == expected; })", "\n".join(keys) + "\n")
  File "/home/mvo/work/cockpit/test/testlib.py", line 208, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))), timeout=self.phantom_wait_timeout)
  File "/home/mvo/work/cockpit/test/testlib.py", line 656, in wait
    return self.run({'cmd': 'wait', 'cond': cond, 'timeout': timeout*1000})
  File "/home/mvo/work/cockpit/test/testlib.py", line 619, in run
    raise Error("timeout")
Error: timeout
```

The `/home/admin/.ssh/identity` key is not loaded into the agent, apparently.  I can't figure this out.  From the journal:

```
syys 30 14:40:04 m10 cockpit-session[1283]: pam_ssh_add: Identity added: /home/admin/.ssh/id_rsa (/home/admin/.ssh/id_rsa)
syys 30 14:40:04 m10 cockpit-session[1283]: pam_ssh_add: Identity added: /home/admin/.ssh/id_ed25519 (/home/admin/.ssh/id_ed25519)
syys 30 14:40:04 m10 cockpit-session[1283]: pam_ssh_add: Failed adding some keys
```

File: test/guest/cockpit-fedora-23.setup [hunk 1]
```diff
@@ -17,7 +17,7 @@ echo 'SELINUX=permissive' > /etc/selinux/config
 # them to not spontaneously change from one test run to the next when
 # the distribution repository is updated.
 #
-COCKPIT_DEPS="json-glib realmd glib-networking libssh selinux-policy-targeted docker pcp-libs pcp kubernetes etcd subscription-manager storaged storaged-lvm2 device-mapper-multipath"
+COCKPIT_DEPS="json-glib realmd glib-networking libssh selinux-policy-targeted docker pcp-libs pcp kubernetes etcd subscription-manager storaged storaged-lvm2 device-mapper-multipath atomic"
```
Comment (dperpeet) [inline test/guest/cockpit-fedora-23.setup]: Can we multiline these deps [like this](https://github.com/cockpit-project/cockpit/blob/64be41182486f0180c07c93946fed1a2f16bcb45/test/guest/cockpit-rhel-7.setup#L47)? It makes things more readable, especially if the dependencies change.

**Comment by petervo** [comment]
Now that #2880 is in a rebase on master should fix the key auth test.

**Comment by stefwalter** [comment]
Needs rebase.

**Comment by stefwalter** [comment]
@dperpeet's review item above isn't fixed yet.

File: test/guest/cockpit-fedora-23.setup (hunk 1 above)
Comment (mvollmer) [inline test/guest/cockpit-fedora-23.setup]: Sure, done.

**Comment by stefwalter** [comment]
Merged, even though there are still problems with Fedora 23, this fixes some of them.

**Pull request closed without merging**